src/sentry/constants.py

```diff
@@ -177,6 +176,14 @@ def get_all_languages():
     'mysql_pwd',
 )
 
+NOT_SCRUBBED_VALUES = set([
```

Comment:
> I originally included None but it's actually already covered by the first clause of sanitize.

doh

> almost by definition if we can think of it and add it to NOT_SCRUBBED_VALUES it's not sensitive!

I'm trying to think of counter arguments here, but I'm coming short. I'd rather not go crazy and start adding a ton of things here though just in case. I think what we have now is a fine start and we can add others if they come up.

> I'm not that familiar with what might get to this point and what shape it would be (for example do you mean 'undefined' the string or some exotic undefined object?)

I mean literally the string value `undefined`. I mentioned this becuase you added `null`.

> What about value being a list/dict etc. Could sanitize be recursive?

It already is recursive on dictionaries, but we don't walk down a list it seems. I could be wrong, but at a glance, this seems to be the behavior.

> I wanted to proof the concept of objects so mundane we didn't want to sanitise them even if their key was "sensitive" before going any further

🥇 

> I'll add tests for auth tomorrow.

🥇 Don't feel pressured to do this though, I'd be happy to do this slightly more annoying part. :)